src/gluonts/evaluation/_base.py

```diff
@@ -345,6 +357,9 @@ def get_metrics_per_ts(
                 pred_target, forecast_quantile
             )
 
+        if self.nan_if_masked_metric:
+            metrics = valmap(nan_if_masked, metrics)
```

Comment:
I wonder, why wouldn't you do it always? What's the benefit of getting back `masked` as a metric?

Reply:
It allows users to differentiate between if the metric calculation resulted in a `nan` or not when looking at the item metrics.
For the purpose of aggregating the metrics it should not matter.

Reply:
Perhaps we should remove `nan_if_masked_metric` instead.

Reply:
Not sure we want to make a differentiation there. I'm for just removing the option and always calling `nan_if_masked`.

@lostella what do you think?